content/post/000-ipfs-in-web-browsers.md

```diff
@@ -0,0 +1,74 @@
+
+Decentralization feels slow in providing user-centric approaches to the internet's biggest problems. Right now true user agency is often too technical, too fragile and too time-consuming to be the default choice.
```

Comment:
```suggestion
Decentralization can feel slow in providing user-centric approaches to the internet's biggest problems. Right now, true user agency is often too technical, too fragile, and too time-consuming to be the default choice.
```

can you explain what you mean by user agency? It seems like a reference to https://en.wikipedia.org/wiki/User_agent - but used in this way it feels like jargon that could use a definition...